packages/contracts-bsc/contracts/common/BEP20.sol

```diff
@@ -40,61 +23,66 @@ import {ClaimableOwnable} from "./ClaimableOwnable.sol";
  *
  * Additionally, an {Approval} event is emitted on calls to {transferFrom}.
  * This allows applications to reconstruct the allowance for all accounts just
- * by listening to said events. Other implementations of the EIP may not emit
+ * by listening to said events. Other implementations of the TIP may not emit
  * these events, as it isn't required by the specification.
  *
  * Finally, the non-standard {decreaseAllowance} and {increaseAllowance}
  * functions have been added to mitigate the well-known issues around setting
- * allowances. See {IERC20-approve}.
+ * allowances. See {IBEP20-approve}.
  */
-abstract contract ERC20 is ClaimableOwnable, Context, IERC20 {
+abstract contract BEP20 is ClaimableOwnable, Context, IBEP20 {
     using SafeMath for uint256;
-    using Address for address;
 
     /**
      * @dev Returns the name of the token.
      */
-    function name() public virtual pure returns (string memory);
+    function name() public view virtual override returns (string memory);
```

Comment:
I know the new `IBEP20.sol` that you imported specifies this as a view function, but changing this line from `pure` to `view` makes this function signature slightly less restrictive than it could be.

If switching back wouldn't break interoperability with other BNB contracts, then I would recommend keeping this as `pure` and updating the `IBEP20.sol` in this repo to be `pure`. We know that, once deployed, we don't anticipate these functions to ever change. Ditto with `symbol()` and `decimals()`.